vm-virtio/src/balloon.rs

```diff
@@ -51,6 +51,8 @@ const DEFLATE_QUEUE_EVENT: DeviceEventT = 2;
 const KILL_EVENT: DeviceEventT = 3;
 // The device should be paused.
 const PAUSE_EVENT: DeviceEventT = 4;
+// The device should update memory.
+const UPDATE_MEMORY_EVENT: DeviceEventT = 5;
```

Comment:
Please remove this patch as this is not needed. The updated `GuestMemoryMmap` is contained inside the `GuestMemoryAtomic<GuestMemoryMmap>` object. This means it is up to date whenever the `virtio-balloon` device accesses the memory regions.